go/libraries/doltcore/sqle/dtables/diff_table.go

```diff
@@ -890,44 +876,62 @@ func GetDiffTableSchemaAndJoiner(format *types.NomsBinFormat, fromSch, toSch sch
 		if err != nil {
 			return nil, nil, err
 		}
+
 		diffTableSchema = j.GetSchema()
-		colCollection = diffTableSchema.GetAllCols()
-		colCollection = colCollection.Append(
+		fullDiffCols := diffTableSchema.GetAllCols()
+		fullDiffCols = fullDiffCols.Append(
 			schema.NewColumn(diffTypeColName, schema.DiffTypeTag, types.StringKind, false),
 		)
-		diffTableSchema = schema.MustSchemaFromCols(colCollection)
+		diffTableSchema = schema.MustSchemaFromCols(fullDiffCols)
 	}
 
 	return
 }
 
-// CalculateDiffSchema returns the schema for the dolt_diff table based on the schemas from the from and to tables.
-// Either may be nil, in which case it will be missing from the resulting schema.
-func CalculateDiffSchema(fromSch, toSch schema.Schema) (schema.Schema, error) {
-
-	colCollection := schema.NewColCollection()
-	if fromSch != nil {
-		colCollection = fromSch.GetAllCols()
+// Convert schemas to ColCollections. One argument must be non-nil. If one is null, the result will be the columns
```

Comment:
(minor) Godoc style for function or member docs requires that the first word of the comment string be the function or member name. 

```suggestion
// ResolveSchemaCollections converts schemas to ColCollections. One argument must be non-nil. If one is null, the result will be the columns
```

Reply:
You can probably also lowercase this function – that'll keep it from being exported out of the package it lives in.

Reply:
"resolve" also has a pretty specific meaning when used (e.g. "resolving unqualified tables or columns"), so it might be better to pick a less loaded term, like `generateDiffSchemas` or something.